karaf/apache-brooklyn/src/main/resources/etc/org.apache.brooklyn.classrename.cfg

```diff
@@ -0,0 +1,21 @@
+################################################################################
```

Comment:
@tbouron On second thoughts lets not hold up this PR changing the feature as there are other config files that do not follow this pattern 

@svet What do you think about creating a common maven artifact that contains all the additional brooklyn config so downstream projects can include them easily

eg `default.catalog.bom`, `org.apache.brooklyn.classrename.cfg`, `org.apache.brooklyn.core.catalog.bomscanner.cfg`, `org.apache.brooklyn.osgilauncher.cfg`